### ISSUE TITLE
Add support for beta features in spatialite models

### ISSUE THREAD

**Issue by elisallenens** [nens/threedi-modelchecker#293]
How I see this:
- A central configuration file should exist, which contains a list of all columns and tables which are beta features, specifying if they are a column or a table.
- A check is added to config.py which goes through every column in config.py and every column in every table in config.py, and, if one of them is filled in, raises an error.
- A flag should be added to the modelchecker to allow ignoring this error.

@jpprins1 @martijn-siemerink what do you think

**Comment by jpprins1** [comment]
Yes, think this sums it up. Maybe call it something like beta_features.py or similar?

**Comment by elisallenens** [comment]
What decides if the features go from beta to stable? A new schema release, or a new modelchecker release? If it's the former, perhaps beta_features.py should be in threedi-schema.
